CHANGELOG.md

```diff
@@ -58,6 +58,11 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 * **The `type_list` header has moved:**
   If you included `<seqan3/core/type_list.hpp>` you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
 
+#### Input/Output
```

Comment:
```suggestion
#### I/O
```
:nail_care: Since this is the actual directory name ?

Reply:
There was already a section `Input/Output` in the file above so I wanted to keep it consistent. I changed both to `I/O` now.